Mage.Sets/src/mage/cards/l/LethalScheme.java

```diff
@@ -0,0 +1,176 @@
+        super(card);
+    }
+
+    @Override
+    public LethalScheme copy() {
+        return new LethalScheme(this);
+    }
+}
+
+// Based loosely on "Venerated Loxodon" and "Change of Plans"
+class LethalSchemeEffect extends OneShotEffect {
+
+    public LethalSchemeEffect() {
+        super(Outcome.Benefit);
+        this.staticText = "Each creature that convoked Lethal Scheme connives.";
+    }
+
+    public LethalSchemeEffect(final LethalSchemeEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public LethalSchemeEffect copy() {
+        return new LethalSchemeEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        LethalSchemeWatcher watcher = game.getState().getWatcher(LethalSchemeWatcher.class);
+        if (watcher != null) {
+            MageObjectReference mor = new MageObjectReference(source.getSourceId(), game);
+            Set<MageObjectReference> creatures = watcher.getConvokingCreatures(mor);
+            if (creatures != null) {
+                Set<Pair<UUID,Permanent>> playerPermanentsPairs =
+                    creatures
+                        .stream()
+                        .map(creatureMOR -> creatureMOR.getPermanentOrLKIBattlefield(game))
+                        .filter(Objects::nonNull)
+                        .map(permanent -> new Pair<>(permanent.getControllerId(),permanent))
+                        .collect(Collectors.toSet());
+
+                Map<Player, Set<Permanent>> permanentsPerPlayer = new HashMap<>();
+
+                playerPermanentsPairs.forEach(pair -> {
+                    Player player = game.getPlayer(pair.getKey());
+                    if(!permanentsPerPlayer.containsKey(player)){
+                        permanentsPerPlayer.put(player, new HashSet<>());
+                    }
+                    permanentsPerPlayer.get(player).add(pair.getValue());
+                });
+
+                if (playerPermanentsPairs.isEmpty()) {
+                    return false;
+                }
+
+                for (Player player : game
+                    .getState()
+                    .getPlayersInRange(source.getControllerId(), game)
+                    .stream()
+                    .map(game::getPlayer)
+                    .filter(Objects::nonNull)
+                    .filter(player -> permanentsPerPlayer.containsKey(player))
+                    .collect(Collectors.toList())) {
+
+                    Set<Permanent> permanents = permanentsPerPlayer.get(player);
+
+                    while (permanents.size() > 0) {
+                        Choice choiceForThisLoop = new ChoiceImpl(true);
+                        choiceForThisLoop.setMessage("Choose next connive to resolve.");
+
+                        permanents.stream()
+                                .forEach(permanent -> choiceForThisLoop.getChoices().add(permanent.getIdName()));
+
+                        player.choose(Outcome.Neutral, choiceForThisLoop, game);
```

Comment:
I am worried that this might through a null pointer exception.

What happens if the player leaves the game or dies (e.g. drawing from an empty library)?